src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -15,6 +15,26 @@ Imports Xunit
 
 Friend Module CompilationUtils
 
+    Public Function CreateCompilation(trees As IEnumerable(Of SyntaxTree),
```

Comment:
If this is a helper with a very special behavior, the name should reflect that and the comment should clearly explain what is special about it.